pyproject.toml

```diff
@@ -35,11 +35,11 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Physics",
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
-dependencies = ["ase>=3.23.0", "joblib", "phonopy", "pymatgen"]
+dependencies = ["ase>=3.23.0", "joblib", "phonopy", "pymatgen", "numpy<2.0.0"]
```

Comment:
which part of `matcalc` is `numpy` v2 incompatible?

Reply:
Hi @janosh, thanks for asking! I think the problem comes from the matgl side, which is currently not fully compatible with numpy v2.0.0 or above. Therefore I set the upper bound for numpy to be safe. Please remove it if needed!

Reply:
happy to leave it for now. just curious